cantabular/contract.go

```diff
@@ -225,9 +225,10 @@ type GetParentAreaCountResult struct {
 }
 
 type GetBlockedAreaCountResult struct {
-	Passed  int `json:"passed"`
-	Blocked int `json:"blocked"`
-	Total   int `json:"total"`
+	Passed         int     `json:"passed"`
+	Blocked        int     `json:"blocked"`
+	Total          int     `json:"total"`
+	TableLeveError *string `json:"table_level_error,omitempty"`
```

Comment:
I don't think this needs to be a pointer? Omitempty should work as expected with an empty string. Also `TableError` might be enough rather than `TableLevelError`, trying to keep names as concise as we can

Reply:
Will check with FE developer if they will prefer the field to be missing or have null in it and change accordingly